src/sql/workbench/browser/modelComponents/viewBase.ts

```diff
@@ -108,8 +108,11 @@ export abstract class ViewBase extends AngularDisposable implements IModelView {
 		let childDescriptor = this.modelStore.getComponentDescriptor(itemConfig.componentShape.id);
 		this.queueAction(containerId, (component) => {
 			this.logService.debug(`Removing component ${itemConfig.componentShape.id} from container ${containerId}`);
-			component.removeFromContainer(childDescriptor);
+			if (component.removeFromContainer) {
```

Comment:
You were seeing this be undefined? That doesn't make any sense - the component should always be a container and thus have this function

Reply:
I'm able to repro this too - let me dig into why this is happening

Reply:
https://github.com/microsoft/azuredatastudio/pull/13478